src/state/pools/index.ts

```diff
@@ -224,58 +223,58 @@ export const fetchPoolsStakingLimitsAsync = () => async (dispatch, getState) =>
   }
 }
 
-export const fetchPoolsUserDataAsync =
-  (account: string): AppThunk =>
-  async (dispatch) => {
-    try {
-      const [allowances, stakingTokenBalances, stakedBalances, pendingRewards] = await Promise.all([
-        fetchPoolsAllowance(account),
-        fetchUserBalances(account),
-        fetchUserStakeBalances(account),
-        fetchUserPendingRewards(account),
-      ])
-
-      const userData = poolsConfig.map((pool) => ({
-        sousId: pool.sousId,
-        allowance: allowances[pool.sousId],
-        stakingTokenBalance: stakingTokenBalances[pool.sousId],
-        stakedBalance: stakedBalances[pool.sousId],
-        pendingReward: pendingRewards[pool.sousId],
-      }))
-
-      dispatch(setPoolsUserData(userData))
-    } catch (error) {
-      console.error('[Pools Action] Error fetching pool user data', error)
```

Comment:
should we have `fetchPoolsUserDataAsync.rejected` reducer?

Reply:
Done